drools-ansible-rulebook-integration-api/src/test/java/org/drools/ansible/rulebook/integration/api/SelectAttrTest.java

```diff
@@ -539,4 +539,98 @@ public void testSelectAttrWithScientificNotation() {
 
         rulesExecutor.dispose();
     }
+
+    @Test
+    public void testSelectAttrForArrayInArray() {
+
+        String JSON_ARRAY_IN_ARRAY =
+                """
+                {
+                  "rules":[
+                    {
+                      "Rule":{
+                        "name":"r1",
+                        "condition":{
+                          "AllCondition":[
+                            {
+                              "SelectAttrExpression":{
+                                "lhs":{
+                                  "Event":"incident.alerts.tags"
```

Comment:
At the moment, this rule doesn't match because `"Event":"incident.alerts.tags"` doesn't treat `alerts` as an array. If we change it to `"Event":"incident.alerts[1].tags"`, the rule would match.